src/globals.ts

```diff
@@ -20,6 +20,8 @@ export const DEFAULTS = {
 
   PINATA_API_KEY: process.env.PINATA_API_KEY,
 
-  PINATA_API_SECRET: process.env.PINATA_API_SECRET
+  PINATA_API_SECRET: process.env.PINATA_API_SECRET,
+
+  APP_URL: process.env.VUE_APP_APP_URL ? process.env.VUE_APP_APP_URL: "https://app.gmneedhelp.xyz"
```

Comment:
you may want a trailing slash here?

Reply:
yep! added, thanks for catching

Reply:
it worked without it either way since the ticket detail link adds a pre-slash but for consistency with other app setups, adding it here too